aosp.json

```diff
@@ -431,19 +427,18 @@
     {
         "id": "com.android.managedprovisioning",
         "label": "Work Setup",
-        "description": "Work Setup/Work profile setup\nManages Android user account profiles.\nThe typical use-case is setting up a corporate profile that is controlled by the employer on an employee's personal device, to keep personal and work data separate.",
+        "description": "Work Setup/Work profile setup\nManages Android user account profiles.\nThe typical use-case is setting up a corporate profile that is controlled by the employer on an employee's personal device, to keep personal and work data separate.\n",
         "web": [
             "https://support.google.com/work/android/answer/6191949",
             "https://developers.google.com/android/work/requirements/work-profile"
         ],
-        "removal": "caution",
-        "warning": "Needed for sandbox apps such as Shelter or Insular/Island."
```

Comment:
Not needed for sandbox apps?

Reply:
I misunderstood about this package. Can't test on my device since I don't have access to a PC (travelling).
Reverting the changes.